fdbserver/storageserver.actor.cpp

```diff
@@ -826,6 +845,13 @@ struct StorageServer {
 		// fallback).
 		Counter quickGetValueHit, quickGetValueMiss, quickGetKeyValuesHit, quickGetKeyValuesMiss;
 
+		Counter kvBytesScan;
```

Comment:
define the semantics of added counters in comment

Reply:
Comments added.